java/pom.xml

```diff
@@ -16,6 +16,7 @@
         <maven.compiler.target>8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <fastffi.revision>0.1.2</fastffi.revision>
+        <arrow.version>13.0.0</arrow.version>
```

Comment:
Does this have to be set? If user install other version of arrow,  the program still work as expect?

Reply:
if work as expected, 13.0.0 is ok